src/pages/404.js

```diff
@@ -1,15 +1,41 @@
-import React from 'react';
+import React from 'react'; //, { useEffect, useState }
 import { Link } from 'gatsby';
 
 import Seo from '../components/seo';
+import ThreeScene from '../components/three-scene';
 
 const Page = () => {
+  // const [isLoading, setIsLoading] = useState(true);
```

Comment:
This is just for development Hot reloading on the dashboard SSR page is a little clunky, even when using `DEV_SSR `